### PR TITLE
Clear up Event and News views.

### PR DESCRIPTION
**Pull request by davidwessman**
Event: removing admin show view, adding admin event controller spec, adding event helper

### CODE REVIEW

File: app/controllers/admin/events_controller.rb [hunk 1]
```diff
@@ -29,6 +29,7 @@ def create
   end
 
   def update
+    @event = Event.find(params[:id])
     if @event.update(event_params)
       redirect_to edit_admin_event_path(@event), notice: alert_update(Event)
     else
```
Comment (jforberg) [inline app/controllers/admin/events_controller.rb]: render action: :edit. Prefer to send error status such as 422 Unprocessable entity.

**Comment by jforberg** [comment]
When rendering form errors, remember to send an error code such as 422 Unprocessable entity.

Otherwise +1

**Comment by davidwessman** [comment]
@jforberg Looking ok? 😄

**Comment by jforberg** [comment]
Yes

**Pull request merged**